src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepository.java

```diff
@@ -103,4 +104,15 @@ List<Envelope> findRecentEnvelopes(
     List<Envelope> getCompleteEnvelopesFromContainer(
         @Param("container") String container
     );
+
+    @Modifying(clearAutomatically = true)
+    @Query("UPDATE Envelope e SET e.classification = 'EXCEPTION', e.status = 'UPLOADED' \n"
+            + "WHERE e.id = :id "
+            + "AND e.container = :container"
```

Comment:
I suggest to add more conditions here e.g. 

AND e.classification = 'SUPPLEMENTARY_EVIDENCE'
AND e.status = '...'

in order to prevent occasional changes